dot_calling.ipynb

```diff
@@ -0,0 +1,2181 @@
+{
```

Comment:
<p>move CLI to end?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/open2c/open2c_examples/pull/8/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='7'/>